src/org/zaproxy/zap/extension/accessControl/ExtensionAccessControl.java

```diff
@@ -1,21 +1,21 @@
 /*
  * Zed Attack Proxy (ZAP) and its related class files.
- * 
+ *
  * ZAP is an HTTP/HTTPS proxy for assessing web application security.
- * 
+ *
  * Copyright 2014 The ZAP Development Team.
- * 
- * Licensed under the Apache License, Version 2.0 (the "License"); 
- * you may not use this file except in compliance with the License. 
- * You may obtain a copy of the License at 
- * 
- *   http://www.apache.org/licenses/LICENSE-2.0 
- *   
```

Comment:
White space changes (throughout the PR) should be reverted. 
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting

This is probably due to IntelliJ trying to be helpful.